Homework_2.playground/Contents.swift

```diff
@@ -0,0 +1,84 @@
+class Stack {
+    private var head: Frame?
```

Comment:
Ok, then I have following concern:

It's hard to understand from the code where I store stack items.
I would prefer to see smith like:

```suggestion
    private var linkedList: LinkedList
```

The `head` is a part of the linked list, not a `Stack`, so this is a bit weird to work with `head` here.

In case of array you don't have such code, do you?
```suggestion
    private var index: ArrayElement?
```

Does that make sense?